101-machine-learning-linkedserivce-create/azuredeploy.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "amlWorkspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the AML workspace"

```

Comment:
does this need to be globally unique?  or if it needs to already exist (according to the prereqs) let's provide some more instruction there, e.g.

Name of the existing AML workspace to create the linked service..........

Reply:
thanks for the suggestion, will update the description.